File: go_service/dao/service_detail.go
package dao

type ServiceDetail struct {
	Info          *ServiceInfo   `json:"info,omitempty" description:"基本信息"`
	HTTPRule      *HttpRule      `json:"http_rule,omitempty" description:"http_rule"`
	TCPRule       *TcpRule       `json:"tcp_rule,omitempty" description:"tcp_rule"`
	GRPCRule      *GrpcRule      `json:"grpc_rule,omitempty" description:"grpc_rule"`
	LoadBalance   *LoadBalance   `json:"load_balance,omitempty" description:"load_balance"`
	AccessControl *AccessControl `json:"access_control,omitempty" description:"access_control"`
}

File: java_service/src/main/java/com/kyriexu/model/ServiceDetail.java
package com.kyriexu.model;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author KyrieXu
 * @since 2021/3/23 20:20
 **/
@JsonInclude(JsonInclude.Include.NON_NULL)
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ServiceDetail {
    private ServiceInfo info;
    private HttpRule httpRule;
    private TcpRule tcpRule;
    private GrpcRule grpcRule;
    private LoadBalance loadBalance;
}
